tests/test_linkedheaders.py

```diff
@@ -0,0 +1,53 @@
+"""
+These tests verify how the n2y block classes convert notion data into Pandoc
+abstract syntax tree (AST) objects, and then into markdown.
+"""
+from pandoc.types import (
+    Str, Space, Header, Link
+)
+
+from n2y.utils import strip_hyphens
+from n2y.notion_mocks import mock_block, mock_rich_text
+from tests.test_blocks import process_block
+
+
+def mock_header_ast(level, suffix, notion_block):
+    return Header(
+        level,
+        ("", [], []),
+        [
+            Link(
+                ("", [], []),
+                [Str("Heading"), Space(), Str(suffix)],
+                (f'#{strip_hyphens(notion_block["id"])}', ""),
+            )
+        ],
+    )
+
+
+def test_heading_1():
+    notion_block = mock_block("heading_1", {"rich_text": [mock_rich_text("Heading One")]})
+    pandoc_ast, markdown = process_block(notion_block, ["n2y.plugins.linkedheaders"])
+    assert pandoc_ast == mock_header_ast(1, "One", notion_block)
+    assert markdown == f'# [Heading One](#{strip_hyphens(notion_block["id"])})\n'
```

Comment:
What's the benefit of using what you've got over this:

```
f'# [Heading One]({notion_block["notion_url"]})\n'
```

(or equivalent)?

Reply:
Ah, I like that. I didn't see "url" in the mocked notion block dict but perhaps we should add it in?

Reply:
Ahh, I thought the url was included in the data from Notion. If it is and it's missing, I think we should add it. If it's not included in the payload from Notion, then we shouldn't and what you have now is great. I definitely want the mocks to match the notion api as best as possible.